src/scopes.js

```diff
@@ -34,6 +34,23 @@ module.exports = function(bookshelf) {
       };
     });
 
+    _.each(['hasMany', 'hasOne', 'belongsToMany', 'morphOne', 'morphMany',
+      'belongsTo', 'through'], function(method) {
+      var original = self.prototype[method];
+      self.prototype[method] = function() {
+        var relationship = original.apply(this, arguments);
+        if (protoProps.scopes && protoProps.scopes.default) {
+          var originalSelectConstraints = relationship.relatedData.selectConstraints;
+          relationship.relatedData.selectConstraints = function(knex, options) {
```

Comment:
Wouldn't this hang the default scope from the originating model, not the model of the related data?